ads/index.js

```diff
@@ -73,6 +74,9 @@ function onAdsComplete (event) {
 			utils.log.info('Ad loaded in slot', event);
 			if (slotsRendered === 0) {
 				perfMark('firstAdLoaded');
+				if (!/spoor-id=0/.test(document.cookie)) {
```

Comment:
Turns out there's a mistake here and actually the "poor people with spoor-ids starting with 0" are specifiacally *not* getting targeted. Fix [here](https://github.com/Financial-Times/n-ui/pull/104)